### PR TITLE
Allow to change DNS resolution from nip.io

### PR DESCRIPTION
**Pull request by andresmgot**
`nip.io` is currently down, causing the integration tests to fail. We should not force to use that choice for all the functions so this PR allows to change that.

I changed the DNS resolution from `nip.io` to `xip.io` in the existing examples so we fix integration tests.

### CODE REVIEW

File: examples/http-custom-path/README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ Trigger:  HTTP
 Dependencies:
 ```
 
+Note that if you don't specify a hostname in your `serverless.yaml` it will be configured to use a DNS resolutor like [`nip.io`](http://nip.io) setting the property `defaultDNSResolution` in the provider section. You can also change the default DNS resolutor to a different service like [`xip.io`](http//xip.io).
```
Comment (sebgoa) [inline examples/http-custom-path/README.md]: A DNS service ( I don't think resolutor is an english word)

**Comment by sebgoa** [comment]
small wording change needed but then /lgtm

**Comment by andresmgot** [comment]
thanks for taking a look

**Comment by andresmgot** [comment]
closing this PR since the changes were merged into #105

**Pull request merged**